playbooks/tasks/iworx-php-scl.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+- name: Enable Interworx SCL PHP Support
+  command: >
+    nodeworx
+    -un
+    -c Http
+    -a enableMultiplePhp
+
+- name: Enable PHP Versions + Set Default Version
+  command: >
+    nodeworx
+    -un
+    -c Http
+    -a multiplePhpOptions
+    --enabled_php_versions /opt/remi/php56,/opt/remi/php70,/opt/remi/php71
+    --default_php_version /opt/remi/php71
```

Comment:
Ditto here - just auto detect the "highest" version from the previous step and put it here.